.circleci/behat-test.sh

```diff
@@ -39,14 +33,31 @@ export WORDPRESS_USER_NAME=$ADMIN_USERNAME
 # Use a generic Pantheon user for testing
 export ADMIN_USERNAME='pantheon-ci-testing'
 
-# Setup the WordPress admin user
-terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- user delete $ADMIN_USERNAME --yes
-{
-  terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- user create $ADMIN_USERNAME no-reply@getpantheon.com --user_pass=$ADMIN_PASSWORD --role=administrator
-} &> /dev/null
+# Check if an admin user with our desired username exists
+ADMIN_USER_EXISTS=$(terminus -n wp ${TERMINUS_SITE}.${TERMINUS_ENV} -- user list --login=${ADMIN_USERNAME} --format=count)
 
-# Set Behat variables from environment variables
+# If so, delete the existing admin user
+if [[ "$ADMIN_USER_EXISTS" == "1" ]]
+then
+  terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- user delete $ADMIN_USERNAME --yes
```

Comment:
Did you intend to delete the admin user before making a backup of the site? Perhaps your intention is to not have an admin user on the test multidevs that are left sitting around? That seems inconvenient to me, but I'm okay with leaving it like this if it's what you intend. A comment about your intentions would be helpful, so that folks can know the impact of changing the way this works if they have different preferences.

Reply:
Yes, that was intentional. The admin user is deleted, if it exists, and recreated on each CI run. I do not want to have lingering, additional admin users on the multidev.